library/Fission/CLI/Command/Up.hs

```diff
@@ -82,13 +92,15 @@ up Up.Options {..} = do
       logDebug $ "Starting single IPFS add locally of " <> displayShow absPath
      
       IPFS.addDir ignoredFiles absPath >>= putErrOr \cid -> do
-        sendRequestM (updateApp url cid copyFiles) >>= \case
-          Left err ->
-            CLI.Error.put err "Server unable to sync data"
-           
-          Right _  -> do
-            CLI.Success.live cid
-            CLI.Success.dnsUpdated url
+          updateApp url cid copyFiles
+            |> retryOnErr [status502, status504] 100 
```

Comment:
It feels like this indentation is kinda wonky but it's the only way that it would parse correctly

Reply:
@dholms This comment is outdated, but the way to make the notation consistent here is:

```haskell
copyFiles
  |> updateApp url cid
  |> retryOnErr [status502, status504] 100 
```

The 2-space indentation is _required_ in `do` notation, since every line is sugar for `>>=`. It is consistent this way.

The other way to write this is a `let` or `where`:

```haskell
let req = updateApp url cid copyFiles
retryOnErr [status502, status504] 100 req

--

foo = retryOnErr [status502, status504] 100 req
  where
    req = updateApp url cid copyFiles
```

I actually think that this reads better than pipes, since we're not "just" transforming data.